src/Sylius/Bundle/ApiBundle/Serializer/Denormalizer/ProductAttributeValueDenormalizer.php

```diff
@@ -51,6 +51,7 @@ public function denormalize(mixed $data, string $type, ?string $format = null, a
 
```

Comment:
Skip custom processing if attribute is missing.
```suggestion
        if (!isset($data['attribute'])) {
            return $this->denormalizer->denormalize($data, $type, $format, $context);
        }
```